src/skin/legacyskinparser.cpp

```diff
@@ -1112,6 +1112,8 @@ QWidget* LegacySkinParser::parseSpinny(const QDomElement& node) {
         return dummy;
     }
     commonWidgetSetup(node, spinny);
+    // Pixmaps are rendered from SVGs upon resizing, so initialize the pixmaps
+    spinny->resize(spinny->size());
```

Comment:
I tried calling it in WSpinny::setup(), but that didn't work either. IIRC WSpinny's initial size was not set up at that point, but I'll try again.